scripts/components/AddFishButton.tsx

```diff
@@ -1,5 +1,6 @@
 import React = require("react");
+import {AddButtonProps} from "../libs/interfaces";
 
-export const AddFishButton = ({text = "default", isAvailable = true, addToOrder}) => (
-  <button disabled={!isAvailable} onClick={addToOrder}>{text}</button>
+export const AddFishButton = (props: AddButtonProps) => (
```

Comment:
once you get more then a few props, I think I like this style better imo. alternative method can be seen https://github.com/ryanluker/typed-catch-of-the-day/pull/26/files#diff-0cffd13014f9e4db7f472bc7742fb0c6R5